Tests/KituraNIOTests/ClientRequestTests.swift

```diff
@@ -151,7 +163,8 @@ extension ClientRequestTests {
              ("testClientRequestAppendsPort", testClientRequestAppendsPort),
              ("testClientRequestSet", testClientRequestSet),
              ("testClientRequestParse", testClientRequestParse),
-             ("testClientRequestBasicAuthentcation", testClientRequestBasicAuthentcation)
+             ("testClientRequestBasicAuthentcation", testClientRequestBasicAuthentcation),
+             ("testClientRequestSyncBehavior", testClientRequestSyncBehavior)
```

Comment:
nit: can we end this line with a comma (so that future additions of tests have a single line code change)